src/maus/__init__.py

```diff
@@ -40,9 +41,16 @@ def merge_lines_with_same_data_element(ahb_lines: Sequence[AhbLine]) -> DataElem
                 # todo: this is line where there is only a description and nothing else. i hate it
                 # f.e. there is 35001: https://github.com/Hochfrequenz/mig_ahb_utility_stack/issues/21
                 continue
-            result.value_pool[
-                data_element_value_entry.value_pool_entry  # type:ignore[index]
-            ] = data_element_value_entry.name.strip()  # type:ignore[assignment,union-attr]
+            if not data_element_value_entry.ahb_expression:
+                # value pool entries with empty/None AHB expression shall not be included
+                # https://github.com/Hochfrequenz/mig_ahb_utility_stack/issues/38
+                continue
+            value_pool_entry = ValuePoolEntry(
+                edifact_key=data_element_value_entry.value_pool_entry,  # type:ignore[arg-type]
+                meaning=data_element_value_entry.name.strip(),  # type:ignore[assignment,union-attr]
+                ahb_expression=data_element_value_entry.ahb_expression,
+            )
+            result.value_pool.append(value_pool_entry)  # type:ignore[index]
     else:
         result = DataElementFreeText(
             entered_input=None,
```

Comment:
Nur als Nebenfrage, wofür brauchen wir in der Maus den `entered_input`? Initial kann er ja nur null/None sein und die validiert wird der EdifactSeed oder?

Reply:
ich dachte ich hätte das von dir übernommen. wenn du es nicht brauchst bei der validation, dann mach doch ein ticket auf und ich schmeiß es im nachgang in einem separaten PR wieder weg ;) auch ok

Reply:
Okay, dann denke ich nochmal in Ruhe drüber nach :D